src/Workspaces/Remote/ServiceHub/Services/BrokeredServiceBase.cs

```diff
@@ -99,5 +117,38 @@ internal static async ValueTask RunServiceImplAsync(Func<CancellationToken, Valu
                 throw ExceptionUtilities.Unreachable;
             }
         }
+
+        private static void SetNativeDllSearchDirectories()
```

Comment:
was this code that we already had, but movd around?

Reply:
Yes, this is existing code.